### PR TITLE
Don't append .ig.local to hostnames

### PR DESCRIPTION
**Pull request by hasegeli**

### CODE REVIEW

**Comment by HerrSpace** [comment]
As we put this on github I think it makes sense to remove this bit. Are we moving forward with this?

**Comment by HerrSpace** [comment]
I rebased this with the permission of Emre. Let's get this merged.

File: igvm/commands.py [hunk 1]
```diff
@@ -540,14 +540,8 @@ def _get_vm(hostname, ignore_reserved=False):
...
     """
-    conditions = [ExactMatch(hostname)]
-    if hostname.endswith('.ig.local'):
-        conditions.append(ExactMatch(hostname[:-len('.ig.local')]))
-    else:
-        conditions.append(Startswith(hostname + '.'))
-
     dataset_obj = Query({
-        'hostname': Or(*conditions),
+        'hostname': Any(hostname, StartsWith(hostname + '.')),
```
[the remark this answers is not in the document]
Comment (hasegeli) [reply igvm/commands.py]: This is just a convenience feature.  It is not a workaround.

**Comment by HerrSpace** [comment]
Retest this please

**Comment by HerrSpace** [comment]
Retest this please

**Comment by HerrSpace** [comment]
Retest this please

**Pull request merged**